src/iuse_actor.cpp

```diff
@@ -3104,3 +3105,41 @@ long place_trap_actor::use( player * const p, item * const it, bool, const tripo
     }
     return 1;
 }
+
+void emit_actor::load( JsonObject &obj )
+{
+    assign( obj, "emits", emits );
+    assign( obj, "scale_qty", scale_qty );
```

Comment:
But that's an item dependent field. If emit struct had it, many uses of it would simply ignore it.
Is it OK to have fields that rely on comments to say things like "only used by items with charges"?